### PR TITLE
Merge installation overview into the homepage

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,10 +1,130 @@
...
 ====================================
 
-!!! note
-    The documentation contained here is currently a work in progress as we migrate our TWiki documentation to GitHub. Please let us know of any issues that you encounter by [opening a ticket](https://ticket.opensciencegrid.org) or e-mailing [goc@opensciencegrid.org](mailto:goc@opensciencegrid.org). Please identify errors, inconsistencies, or incorrect information in the documentation, and also suggest extra content that you feel would be helpful to include in the documentation.
-
-Welcome to the home page of the Open Science Grid (OSG) Site Administrator documentation! If you're a new OSG site adminstrator, start with the [Installation Overview](install-overview). If you are not a site adminstrator...
+Welcome to the home page of the Open Science Grid (OSG) Site Administrator documentation! If you are not a site adminstrator...
 
 - If you are a researcher interested in using OSG resources, you can find user documentation [here](https://support.opensciencegrid.org/support/home). 
 - If you'd like to learn more about the OSG and our mission, visit our website [here](https://www.opensciencegrid.org/).
```
[the remark this answers is not in the document]
Comment (bbockelm) [reply docs/index.md]: Yes, I like these!  Please keep.

File: docs/index.md [hunk 2]
```diff
@@ -1,10 +1,130 @@
...
+
+This document outlines the overall installation process for an OSG site and provides many links into detailed installation, configuration, troubleshooting, and similar pages. If you do not see software-related technical documentation listed here, try the search bar to the left or contacting us at [goc@opensciencegrid.org](mailto:goc@opensciencegrid.org).
+
+Plan the Site
+-------------
+
+If you have not done so already, [plan the overall architecture of your OSG site](site-planning). It is recommended that your plan be sufficiently detailed to include the OSG hosts that are needed and the main software components for each host. Be sure to consider [the operating systems that OSG supports](https://twiki.opensciencegrid.org/bin/view/Documentation/Release3/SupportedOperatingSystems). A simple way to organize this information is in a table; for example, a basic site might include:
```
Comment (bbockelm) [inline docs/index.md]: > A simple way to organize this information is in a table

Maybe I'm just overly sensitive, but this seems borderline insulting of the reader's intelligence.

I don't think the sentence loses content if we just have it be:

> For example, a basic site might include:

File: docs/index.md [hunk 3]
```diff
@@ -1,10 +1,130 @@
...
+| Worker Nodes         | `wNNN.cluster.example.edu`          | OSG worker node client (`osg-wn-client`)                 |
+
+Prepare the Batch System
+------------------------
+
+The assumption is that you have an existing batch system at your site. Currently, we support [HTCondor](http://research.cs.wisc.edu/htcondor/), [LSF](http://www-03.ibm.com/systems/platformcomputing/products/lsf/), [PBS](http://www.pbsworks.com) and [TORQUE](http://www.adaptivecomputing.com/products/open-source/torque/), [SGE](http://en.wikipedia.org/wiki/Oracle_Grid_Engine), and [Slurm](http://slurm.schedmd.com) batch systems.
+
+For smaller sites (less than 50 worker nodes), the most common way to add a site to OSG is to install the OSG Compute Element (CE) on the central host of your batch system.  At such a site - especially if you have minimal time to maintain a CE - you may want to contact goc@opensciencegrid.org to ask about using an OSG-hosted CE instead of running your own.  Before proceeding with an install, be sure that you can submit and successfully run a job from your OSG CE host into your batch system.
```
Comment (bbockelm) [inline docs/index.md]: Turn `goc@osg` into a `mailto:` link.

File: docs/index.md [hunk 4]
```diff
@@ -1,10 +1,130 @@
...
+
+Add OSG Software
+----------------
+
+If necessary, provision all OSG hosts that are in your site plan and that do not exist yet.
+
+!!! note
+    For sites with more than a trivial number of hosts, it is recommended to use some sort of configuration management tool to install, configure, and maintain your site. While beyond the scope of OSG’s documentation to explain how to select and use such a system, some popular configuration management tools are [Puppet](http://puppetlabs.com), [Chef](https://www.chef.io), [Ansible](https://www.ansible.com), and [CFEngine](http://cfengine.com).
```
Comment (bbockelm) [inline docs/index.md]: "trivial number of hosts" -> "handful of worker nodes".  Minor, but "trivial" is in the eye of the beholder.

File: docs/index.md [hunk 5]
```diff
@@ -1,10 +1,130 @@
...
+
+### General Installation Instructions ###
```
Comment (bbockelm) [inline docs/index.md]: The rest of this document turns into a massive link dump, looking a lot like the nav bar (and referencing outdate stuff like OSG 3.1 and OSG 3.2).

Each sub-section should include a sentence or two on which things to read.  Given the emphasis on new admins in the first half of the document, we might want to skew toward new admins here too -- and downplay things like upgrading or esoteric topics for old sites (think: SHA-2 compliance).

**Comment by brianhlin** [comment]
I think there's some value in the link dump until our site-planning document is up to par as it provides indications of optional vs required software and more details compared to the ToC. Though, I think the `General Installation Instructions` as well as the `Installing and Managing Certificates for Site Security` sections can likely be removed if the WN and CE documents reference the appropriate docs.

**Comment by bbockelm** [comment]
Oh yeah - there's a lot of work to do in that section.  It's not that there shouldn't be a list of links of some sort -- but we should be judicious in which ones we present in the introductory text (and we should add more prose).

As I said earlier, I don't think we ought to block a merge on this.  We can save it for later.

**Pull request merged**